### PR TITLE
Mt3 398 other support

### PR DESCRIPTION
**Pull request by LBHABEBIC**
This page is ready without the 'add another' feature. Currently the next page is the Verify page. I'm not sure if this will need to be changed as what shows on balsamiq is a slightly different varify page. 

# What?

# Why?

# Notes

# Next steps

### CODE REVIEW

**Comment by Gweaton** [comment]
Would you be able to squash these 2 commits into one? As the commit message is the same, it'd be good to put them together if you can remember how - I'll show you how to do it tomorrow if you're unsure (reminder: it requires a `git rebase -i`)

File: steps/PageSlugs.ts [hunk 1]
```diff
@@ -116,7 +116,8 @@ export const repeatingStepSlugs = [
   PageSlugs.TenantPhoto,
   PageSlugs.NextOfKin,
   PageSlugs.Carer,
-  PageSlugs.OtherSupport
+  PageSlugs.OtherSupport,
+  PageSlugs.Verify
```
Comment (erbridge) [inline steps/PageSlugs.ts]: I don't think `Verify` belongs here. What motivated adding it?

File: steps/id-and-residency/present-for-check.tsx [hunk 2]
```diff
@@ -35,6 +35,7 @@ const TenantsSelect: React.FunctionComponent<Omit<
   return (
     <Checkboxes
       {...props}
+      required={true}
```
Comment (erbridge) [inline steps/id-and-residency/present-for-check.tsx]:
```suggestion
      required
```

This the same in React.

Comment (erbridge) [reply steps/id-and-residency/present-for-check.tsx]: Also, this change has snuck in from somewhere else. In future, please try to isolate your work to the ticket at hand.

File: storage/Storage.ts [hunk 3]
```diff
@@ -159,6 +158,7 @@ export default class Storage {
             upgrade.createStore("photo");
             upgrade.createStore("nextOfKin");
             upgrade.createStore("carer");
+            upgrade.createStore("otherSupport");
```
Comment (erbridge) [inline storage/Storage.ts]: You need to bump the database version and do this migration in the new version, or it won't be reflected on devices that have already opened this database. Grab me tomorrow if you want to talk it through.

File: __tests__/features/end-to-end.steps.ts [hunk 4]
```diff
@@ -505,6 +510,30 @@ defineFeature(loadFeature("./end-to-end.feature"), test => {
         })
       ).sendKeys(processData.residents[presentTenantRef].carer.notes);
 
+      await browser!.submit();
+      //other support page
```
Comment (erbridge) [inline __tests__/features/end-to-end.steps.ts]:
```suggestion

      // Other support page
```

Definitely a nitpick, but if you're here...

File: steps/PageSlugs.ts (hunk 1 above)
Comment (LBHABEBIC) [reply steps/PageSlugs.ts]: i think these have snuck in somehow from another branch. Sorry.

**Review by Gweaton — APPROVED** [review]

**Pull request merged**